test/integration/graphql/customer.test.js

```diff
@@ -87,6 +87,36 @@ describe('Query.customer', () => {
     })
   })
 
+  it('should handle error', async () => {
+    await mockServer.server.mock.useRouteVariant(
```

Comment:
It would be really good to get smoke tests hitting the actual service in early environments to confirm we get response codes we expect, as we are relying on specific behaviour quite a bit.

Like if V1 started returning proper JSON errors our health check wouldn't flag it correctly.

Reply:
you want to run these unit tests agains environments? I guess we could, but can we add this as a seperate ticket? Doesnt really fall into the remit of this work

Reply:
Yes definitely a separate piece lets raise a ticket for this to be part of smoke tests.